src/components/SearchResultsGrid/SearchResultsGrid.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { SearchResultsGrid } from '../index';
+
+describe('Navbar', () => {
+  it('displays background message when no results where found', () => {
```

Comment:
nice test, you can also provide some mock data to this and test if video cards are correctly rendering, because that is the main functionality